INSTALL.md

```diff
@@ -1,8 +1,16 @@
-### Installation instructions 
+## Installation instructions 
 
 Please follow the following steps in order to set up your development
 environment.
 
+### Docker
+
+1. Make sure [docker](https://www.docker.com/) is installed.
+2. Run `docker build -t bench-routes .`
+3. Run `docker run -p 9090:9090 -it bench-routes`
+
+### Local machine
+
 1. Install Golang +1.10.x (assuming `git` is already installed).
 2. Make a default repository for cloing the project. This should be strictly inside the `GOPATH`. Paste this instruction in your terminal to get started.
 `mkdir -p $GOPATH/src/github.com/zairza-cetb/`.
```

Comment:
could you please update the installation instructions with `go mod vendor` so that the users could get the dependencies in the vendor folder.

Reply:
@Harkishen-Singh Hmm, using the go modules that I added replaces the `vendor/` directory, and thus is no longer necessary.